collector_uncompressed.go

```diff
@@ -0,0 +1,242 @@
+package ftdc
+
+import (
+	"bytes"
+	"io"
+
+	"github.com/mongodb/ftdc/bsonx"
+	"github.com/mongodb/mongo-go-driver/bson"
+	"github.com/pkg/errors"
+)
+
+// NewUncompressedCollectorJSON constructs a collector that resolves
+// data into a stream of JSON documents. The output of these
+// uncompressed collectors does not use the FTDC encoding for data,
+// and can be read as newline seperated JSON.
+//
+// This collector will not allow you to collect documents with
+// different schema (determined by the number of top-level fields,)
+// but does no special handling of schema changes.
```

Comment:
This line seems funny: the first part of the sentence makes it clear that this does not do anything special with schema changes because you do not allow the collection of documents with a different schema. Instead I would just add that it returns an error when adding a document with a different schema.